lib/team_api/api_impl.rb

```diff
@@ -32,6 +33,15 @@ def generate_index_endpoint(endpoint, title, description, items)
       }
     end
 
+    def generate_schema_endpoint(schema_file_location)
+      return if schema_file_location.nil? || schema_file_location.empty?
```

Comment:
It won't have a `schemas` link at all, so it shouldn't result in a broken URL. Still not great if someone is expecting to find a schema, but at least better than a 404 :)